python/test/test_subprocesses.py

```diff
@@ -7,6 +7,10 @@
     'flake8 linting': 'cd .. && flake8 . --count --show-source --statistics',
     # Had non-reproducible errors between local runs, so disable caching.
     'mypy type checking': 'mypy . --cache-dir=/dev/null',
+    # Looking for cases where first character is not `'`, `"`, or `f`
+    # (for f-string). Negative lookahead would be more precise.
+    '"print" in test/ should be explained': "! egrep 'print\\(([^\"'\"'\"'f]' -r test",
+    '"print" in src/ should be log instead': "! egrep '^\\s+print\\(' -r src",
```

Comment:
These could be done in python, like in the [example data PR](https://github.com/opendp/opendp/pull/2229/files#diff-a2d90aa4999938f1e470e0ed44ad7e58f4ac72e154368d73576ff56aa2fe7e6eR14-R18), but I don't have a strong preference.